plugin/src/main/kotlin/com/github/triplet/gradle/play/GenerateResourcesTask.kt

```diff
@@ -30,6 +35,12 @@ open class GenerateResourcesTask : DefaultTask() {
     private val resSrcDirs: List<File> by lazy {
         variant.sourceSets.map { project.file("src/${it.name}/$PLAY_PATH") }
     }
+    private val defaultLocale by lazy {
+        resSrcDirs.mapNotNull {
+            File(it, AppDetail.DEFAULT_LANGUAGE.fileName).orNull()
+                    ?.readText()?.normalized().nullOrFull()
+        }.lastOrNull()
```

Comment:
Can you explain why `lastOrNull()` is the right option here?

Reply:
The `resSrcDirs` are sorted from lowest to highest priority. So taking the last element means getting the highest priority override—it's basically a more efficient way of reversing the list and searching for the most specific override. FYI, the `resSrcDirs` looks something like this: `main, flavor, buildType, flavorBuildType`.